test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -1201,6 +1201,148 @@ public async void TestPacMan_LegacyPackageRefProjects_UpdatePackage_KeepExisting
             }
         }
 
+        [Fact]
+        public async void DependencyGraphRestoreUtility_LegacyPackageRef_Restore_MissingProjectsInSolution()
+        {
+            using (var packageSource = TestDirectory.Create())
+            {
+                // Arrange
+                var sourceRepositoryProvider = TestSourceRepositoryUtility.CreateSourceRepositoryProvider(
+                    new List<Configuration.PackageSource>()
+                    {
+                        new Configuration.PackageSource(packageSource.Path)
+                    });
+
+                using (var testSolutionManager = new TestSolutionManager(true))
```

Comment:
Add a named parameter to explain the pesky `true` literal.

Reply:
Named parameter is `foo` which is not even used. I dont want to refactor it with this PR. But we should just delete this parameter. For now, this doesn't make any sense to even make it a named parameter